qa/vagrant/src/main/java/org/elasticsearch/packaging/test/PackagingTestCase.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.packaging.test;
+
+import com.carrotsearch.randomizedtesting.RandomizedRunner;
+import com.carrotsearch.randomizedtesting.annotations.TestMethodProviders;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.lucene.util.LuceneJUnit3MethodProvider;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+
+@RunWith(RandomizedRunner.class)
+@TestMethodProviders({
+    LuceneJUnit3MethodProvider.class
+})
+/**
+ * Class that all packaging test cases should inherit from. This makes working with the packaging tests more similar to what we're
+ * familiar with from {@link org.elasticsearch.test.ESTestCase} without having to apply its behavior that's not relevant here
+ */
+public abstract class PackagingTestCase {
+
+    protected final Log logger = LogFactory.getLog(getClass());
+
+    @Rule
+    public final TestName testNameRule = new TestName();
+
+    @Before
+    public void logTestNameBefore() {
+        logger.info("[" + testNameRule.getMethodName() + "]: before test");
```

Comment:
Is this really necessary? Seems like it will produce a lot of noise.

Reply:
I think we want it because it'll be necessary to make sense of any other logging we do in the tests. The noise won't show up for people running it in the console because of how the vagrant tasks moderate their output - it'll only show up in CI because it runs with `--console plain --info`

Reply:
Do we really need a before and after? These are run completely sequentially, so the "before" of one test is the "after" of the previous. I'm just thinking of what the old output used to look like (a single line per test in most cases with "OK") compared to what we are moving to here (many lines per test, if I understand correctly).

Reply:
Makes sense, I removed the after test logging